notes.txt

```diff
@@ -0,0 +1,31 @@
+This branch adds pm-cpu to maint-1.0, but there are still some changes
+required to get tests running.
+
+There are 2 changes in files under submodules, so will have to work out best way to fix.
+
+These are changes needed with higher version of GNU (which is all that is avail on PM)
```

Comment:
This implies the changes below haven't been done but it looks like some of them have.  Rewrite to describe what was already done vs. what still needs to be done.